src/Veldrid.ImageSharp/ImageSharpCubeMapTexture.cs

```diff
@@ -0,0 +1,105 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+using System;
+using System.Runtime.InteropServices;
+
+namespace Veldrid.ImageSharp
+{
+    public class ImageSharpCubeMapTexture
+    {
+    
+        /// <summary>
+        /// An array of images, each face of a cubemap.
+        /// </summary>
+        public Image<Rgba32>[] CubeMapTextures { get; }
+
+        /// <summary>
+        /// The width of a cubemap texture.
+        /// </summary>
+        public uint Width => (uint)CubeMapTextures[0].Width;
+
+        /// <summary>
+        /// The height of a cubemap texture.
+        /// </summary>
+        public uint Height => (uint)CubeMapTextures[0].Height;
+
+        /// <summary>
+        /// The pixel format cubemap textures.
+        /// </summary>
+        public PixelFormat Format => PixelFormat.R8_G8_B8_A8_UNorm;
+
+        /// <summary>
+        /// The size of each pixel, in bytes.
+        /// </summary>
+        public uint PixelSizeInBytes => sizeof(byte) * 4;
+
+        /// <summary>
+        /// Provides standardized access to the cubemap texture array
+        /// </summary>
+        private const int FRONT = 0;
+        private const int BACK = 1;
+        private const int LEFT = 2;
+        private const int RIGHT = 3;
+        private const int TOP = 4;
+        private const int BOTTOM = 5;
+
+        public ImageSharpCubeMapTexture(string frontPath,

```

Comment:
I think we should rename these so that they don't mention direction (since that is based on coordinate system conventions). Instead, they could be named `positiveXPath`, `negativeXPath`, `positiveYPath`, etc.